convert2rhel/actions/pre_ponr_changes/subscription.py

```diff
@@ -108,17 +108,6 @@ def run(self):
             logger.task("Convert: Get RHEL repository IDs")
             rhel_repoids = repo.get_rhel_repoids()
 
-            logger.task("Convert: Subscription Manager - Check required repositories")
-            non_available_repos = subscription.check_needed_repos_availability(rhel_repoids)
-            if non_available_repos:
-
-                self.add_message(
-                    level="WARNING",
-                    id="NON_AVAILABLE_REPOSITORIES_DETECTED",
-                    message="%s repositories are not available - some packages"
-                    " may not be replaced and thus not supported." % ", ".join(non_available_repos),
-                )
-
```

Comment:
Is this the check that Michal said we could remove now?

Reply:
Yes, it is apart of this ticket https://issues.redhat.com/browse/RHELC-1017